blaze/expr/expressions.py

```diff
@@ -104,8 +101,7 @@ class Expr(Node):
     contains shared logic and syntax.  It in turn inherits from ``Node`` which
     holds all tree traversal logic
     """
-
-    __slots__ = '_hash', '__weakref__'
+    __slots__ = '_hash', '__weakref__', '__dict__'
```

Comment:
With this change, do we lose the benefit of a lower memory footprint from using `__slots__`? It's probably not a huge performance benefit for us right now, I'm just curious.